source/agora/consensus/EnrollmentManager.d

```diff
@@ -211,12 +241,16 @@ public class EnrollmentManager
         assumeSafeAppend(enrolls);
 
         static Enrollment[] pool_enrolls;
-        this.enroll_pool.getEnrollments(pool_enrolls);
+        this.enroll_pool.getEnrollments(pool_enrolls, height);
         foreach (enroll; pool_enrolls)
         {
             const enroll_height = this.getEnrolledHeight(enroll.utxo_key);
+            const avail_height =
+                this.enroll_pool.getAvailableHeight(enroll.utxo_key);
```

Comment:
Why would we need `getAvailableHeight` if we know `getEnrollments` already filters them out ???

Reply:
It's because we need to filter out the enrollment which left unintentionally in the pool although it had become a validator.